src/exports.cpp

```diff
@@ -25,7 +25,7 @@ bool osara_isShortcutHelpEnabled() {
 	return isShortcutHelpEnabled;
 }
 
-void osara_getVersion(char* versionOut, int versionOut_sz) {
+void osara_getVersion(char* versionOut, uintptr_t versionOut_sz) {
```

Comment:
We can't change this to uintptr_t, as this would change how this has to be called from API consumers on 64 bit systems, since uintptr_t is 64 bit there. I guess we'll need to change `_vararg_osara_getVersion(` instead to do `(int)(uintptr_t)` or something like that. I can't quite remember the syntax to tell the compiler "yes, we know we're truncating, we're okay with it".

Reply:
I think I fixed it now.